.evergreen/config.yml

```diff
@@ -199,6 +199,38 @@ functions:
         script: |
           DRIVERS_TOOLS="${DRIVERS_TOOLS}" sh ${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/run-mongohouse-local.sh
 
+  "create serverless instance":
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        shell: bash
+        script: |-
+          ${PREPARE_SHELL}
+          export PROJECT=${PROJECT}
+          export SERVERLESS_DRIVERS_GROUP=${SERVERLESS_DRIVERS_GROUP}
+          export SERVERLESS_API_PUBLIC_KEY=${SERVERLESS_API_PUBLIC_KEY}
+          export SERVERLESS_API_PRIVATE_KEY=${SERVERLESS_API_PRIVATE_KEY}
+          export SERVERLESS_ATLAS_USER=${SERVERLESS_ATLAS_USER}
+          export SERVERLESS_ATLAS_PASSWORD=${SERVERLESS_ATLAS_PASSWORD}
```

Comment:
I assume these environment variables are defined in the Evergreen project config?

Reply:
That is correct, they are also available through the private design document for local use.